public/pages/Rollups/containers/Rollups/Rollups.tsx

```diff
@@ -0,0 +1,583 @@
+          <EuiFlexItem>
+            <EuiTitle size={"m"}>
+              <h3>{"Rollup jobs (" + `${rollups.length}` + ")"}</h3>
+            </EuiTitle>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup alignItems="center" gutterSize="s">
+              <EuiFlexItem grow={false}>
+                <EuiButton disabled={!selectedItems.length} onClick={this.onDisable}>
+                  Disable
+                </EuiButton>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButton
+                  disabled={!selectedItems.length}
+                  onClick={() => {
+                    this.onEnable();
+                  }}
+                >
+                  Enable
+                </EuiButton>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiPopover
+                  id="action"
+                  button={actionButton}
+                  isOpen={isPopoverOpen}
+                  closePopover={this.closePopover}
+                  panelPaddingSize="none"
+                  anchorPosition="downLeft"
+                >
+                  <EuiContextMenuPanel items={actionItems} />
+                </EuiPopover>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButton onClick={this.onClickCreate} fill={true}>
+                  Create rollup job
+                </EuiButton>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+
+        <div style={{ padding: "initial" }}>
+          <EuiFlexGroup style={{ padding: "0px 5px" }}>
+            <EuiFlexItem>
+              <EuiFieldSearch fullWidth={true} value={search} placeholder="Search" onChange={this.onSearchChange} />
+            </EuiFlexItem>
+            {pageCount > 1 && (
+              <EuiFlexItem grow={false} style={{ justifyContent: "center" }}>
+                <EuiPagination
+                  pageCount={pageCount}
+                  activePage={page}
+                  onPageClick={this.onPageClick}
+                  data-test-subj="indexControlsPagination"
+                />
+              </EuiFlexItem>
+            )}
+          </EuiFlexGroup>
+
+          <EuiHorizontalRule margin="xs" />
+
+          <EuiBasicTable
+            columns={rollupsColumns}
+            isSelectable={true}
+            itemId="_id"
+            items={rollups}
+            noItemsMessage={
+              <RollupEmptyPrompt filterIsApplied={filterIsApplied} loading={loadingRollups} resetFilters={this.resetFilters} />
+            }
+            onChange={this.onTableChange}
+            pagination={pagination}
+            selection={selection}
+            sorting={sorting}
+            tableLayout={"auto"}
+          />
+          {isDeleteModalVisible && (
+            <DeleteModal
+              rollupId={selectedItems.length ? this.concatName() : ""}
```

Comment:
Also what does it look like in the modal when you have 50 rollups selected and they are all concated?

Reply:
Made an attempt to see what it would look like when deleting 50 rollup jobs. Doesn't seem too bad but we can still consider if we want to have a size limit such as 20 to the concatName() function to show as "test19, test20, ..."

![image](https://user-images.githubusercontent.com/71157062/99633457-cee77880-29f3-11eb-8904-0c778b0308e0.png)

Reply:
Got it, seems fine.